### ISSUE TITLE
Commant Not Found

### ISSUE THREAD

**Issue by saima68931** [nils-soderman/vscode-sitemap-generator#6]
Whenever I run: Sitemap-Generator: New Sitemap

it shows the error:

zsh: command not found: Sitemap-Generator:

Any solution, please

**Comment by saima68931** [comment]
Only once it generated the sitemap. A few days back after that no update, even the sitemap file was removed automatically

**Comment by jotakar** [comment]
I have the same problem, sitemap-generator not cmdlet not found, so it's impossible generate any sitemap. 
Is the pluggin working?

**Comment by saima68931** [comment]
yes it works...i didnt know how to use it... jus press ctrl+shift+p and
then select regenerate sitemap... if you have setup other settings...best
